src/tkn.ts

```diff
@@ -138,27 +139,32 @@ export class TknImpl implements Tkn {
     const kubectlCheck = RegExp('kubectl:\\s*command not found');
     if (kubectlCheck.test(getStderrString(result.error))) {
       const tknMessage = 'Please install kubectl.';
+      telemetryLogCommand('kubeclt_not_found', tknMessage);
```

Comment:
The name of `telemetryLogCommand` function is worry me, as you are not reporting of execution of command. Maybe we need another function to report such info.